test/services/actionRulesService.test.js

```diff
@@ -0,0 +1,37 @@
+const actionRulesService = require('../../server/services/actionRulesService')
+const actionsService = require('../../server/services/actionsService')
+const actionID = 'FG1'
+const ruleID1 = 3
+const ruleID2 = 5
+const enabled = false
+
+describe('actionRulesService', () => {
+  test('action rules service returns an object', async () => {
+    const rule = await actionRulesService.update(actionID, ruleID1, enabled)
+    expect((rule instanceof Object)).toEqual(true)
+  })
+
+  test('action rules service returns a rule object containing given id and enabled state', async () => {
+    const rule = await actionRulesService.update(actionID, ruleID1, enabled)
+    expect(rule).toEqual(
+      expect.objectContaining({
+        id: ruleID1,
+        enabled: enabled
+      })
+    )
+  })
+
+  test('action rules service changes to rule enabled persist on multiple calls', async () => {
+    await actionRulesService.update(actionID, ruleID1, enabled)
```

Comment:
Should we ensure that at least one rule has a different `enabled` value at the beginning of this test to the one that is set within the test?

Reply:
Updated test, although it has been moved to `rulesService.js` where it should have been originally (as per your comment below).